### PR TITLE
chore: RPC patch

### PR DESCRIPTION
**Pull request by SwenSchaeferjohann**
* bump versions to latest NPM releases
* changes the JS RPC return type for endpoints with cursors. Wraps inside WithCursor<{items, cursor}>, so values need to be accessed like e.g. compressedAccounts.items[0]

### CODE REVIEW

File: js/compressed-token/package.json [hunk 1]
```diff
@@ -1,28 +1,23 @@
 {
     "name": "@lightprotocol/compressed-token",
-    "version": "0.4.1",
+    "version": "0.8.0",
```
Comment (ananas-block) [inline js/compressed-token/package.json:3]: current release is 0.10.1

File: js/stateless.js/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@lightprotocol/stateless.js",
-    "version": "0.5.1",
+    "version": "0.9.0",
```
Comment (ananas-block) [inline js/stateless.js/package.json:3]: current version is 0.10.1

**Review by ananas-block — APPROVED** [review]
Apart from the two incorrect versions lgtm.

File: js/stateless.js/package.json (hunk 2 above)
Comment (ananas-block) [reply js/stateless.js/package.json:3]:
```suggestion
    "version": "0.10.1",
```

File: js/compressed-token/package.json (hunk 1 above)
Comment (ananas-block) [reply js/compressed-token/package.json:3]:
```suggestion
    "version": "0.10.1",
```

**Review by vadorovsky — APPROVED** [review]

**Pull request merged**